app/assets/sass/application.scss

```diff
@@ -6,7 +6,7 @@ $govuk-new-link-styles: true;
 $govuk-assets-path: '/govuk/assets/';
 
 // Import GOV.UK Frontend and any extension styles if extensions have been configured
-@import "lib/extensions/extensions";
+@import ".generated-assets/sass/extensions";
```

Comment:
it'd be good to discuss this a bit more - dotfiles have given out users issues in the past, as has putting generated files in userland ('public') so it'd be good to avoid these issues if we can

Reply:
agreed to call it `.tmp` for now

Reply:
Changed to .tmp as requested